src/components/EventPolicy/EventPolicy.tsx

```diff
@@ -205,11 +211,11 @@ const EventPolicy = () => {
           translationContent={pageContent.addEventForm}
           onConfirm={onSubmit}
           isOpen={
-            !!selectedPolicy && showEventModal && eventToBeEdited
+            showEventModal && !!selectedPolicy && eventToBeEdited
```

Comment:
I'm confused about this condition. Cause `EventModal` will be rendered if `selectedPolicy` is `true` as per line no 207. Then technically `selectedPolicy` should not be part of the condition. Also, `eventToBeEdited` can be used directly in co-relation with `showEventModal`. Ternary condition can be avoided completely in my opinion.

Thoughts?